lib/Zonemaster/Backend/Validator.pm

```diff
@@ -136,14 +195,64 @@ sub untaint_engine_type {
     return _untaint_pat( $value , $ENGINE_TYPE_RE );
 }
 
+sub untaint_mariadb_database {
+    my ( $value ) = @_;
+    return _untaint_pat( $value,  $MARIADB_IDENT_RE, $MARIADB_DATABASE_LENGTH_RE );
+}
+
+sub untaint_mariadb_user {
+    my ( $value ) = @_;
+    return _untaint_pat( $value,  $MARIADB_IDENT_RE, $MARIADB_USER_LENGTH_RE );
+}
+
+sub untaint_password {
+    my ( $value ) = @_;
+    return _untaint_pat( $value,  $PASSWORD_RE );
+}
+
+sub untaint_positive_int {
+    my ( $value ) = @_;
+    return _untaint_pat( $value,  $POSITIVE_INT_RE );
+}
+
+sub untaint_positive_millis {
+    my ( $value ) = @_;
+    return _untaint_pat( $value,  $MILLIS_RE, $POSITIVE_NUM_RE );
```

Comment:
see my comment above on `POSITIVE_NUM_RE`, I'm wondering if this is needed here.